website/files/models/base.py

```diff
@@ -398,9 +399,10 @@ def copy_under(self, destination_parent, name=None):
 
     def move_under(self, destination_parent, name=None):
         self.name = name or self.name
+        if self.parent.node != destination_parent.node:
+            project_signals.file_moved_node.send(self, destination_node=destination_parent.node)
```

Comment:
When you are renaming a file, `name` is the new name of the file. I didn't pass it to the signal since it's not necessary for updating the comments.